_sass/_navbar.scss

```diff
@@ -135,6 +135,7 @@
 			display: block;
 			margin: auto;
 			margin-top: 100px;
+			margin-bottom: 100px;
```

Comment:
Should we change how big each column of the navbar is instead of adding a margin-bottom? Since now the about part of the navbar gets wonky on iPhone X & up
![IMG_0375](https://user-images.githubusercontent.com/65370631/116761839-8930d200-a9cd-11eb-874e-09d084ae9117.jpeg)